### PR TITLE
Slight overhaul of OS icons

### PR DESCRIPTION
**Pull request by TomBrien**
Mainly this changes the Apple icons to include iOS (in most places). This is in prep for adding macOS icons soon (icon itself included here). I've also been through he docs and tried to make the icon use and layout more consistent.

### CODE REVIEW

File: docs/notifications/actionable.md [hunk 1]
```diff
@@ -146,7 +147,7 @@ When an action is selected an event named `ios.notification_action_fired` for iO
 }
 ```
 
-![android](/assets/android.svg) Android event example
+![Android](/assets/android.svg) &nbsp; Android event example
```
Comment (dshokouhi) [inline docs/notifications/actionable.md:150]: Was this in error? :smile:

Comment (TomBrien) [reply docs/notifications/actionable.md:150]: Nope. There's a few of those. I changed all the Android ones to have a capital in the alt text. Some did and some didn't which meant not all were inverting properly in dark mode (needs to be consistent for the CSS or I need to add a second rule). 

`&nbsp` means non-breaking space. I used it to force an extra space between the logo and "Android" as when the logo was used in a sentence as it were it looked very squashed.

Comment (dshokouhi) [reply docs/notifications/actionable.md:150]: Thanks for explaining :)

Comment (TomBrien) [reply docs/notifications/actionable.md:150]: No worries. Dosusaurus had a change which makes it really hard to elegantly do the invert. Currently I'm doing it a very hacky way. 

Anyway, if you're happy with this all free to merge :) just wanted to make sure you'd seen as is a largish diff.

**Review by dshokouhi — APPROVED** [review]
Looks good to me

**Comment by TomBrien** [comment]
Cheers

**Pull request merged**